src/Helper/ValidateStreet.php

```diff
@@ -60,14 +60,15 @@ public static function validate(string $fullStreet, string $localCountry, ?strin
      */
     public static function getStreetRegexByCountry(string $local, string $destination): ?string
     {
-        if (
-            ($local === AbstractConsignment::CC_BE && $destination === AbstractConsignment::CC_NL) ||
-            ($local === AbstractConsignment::CC_NL && $destination === AbstractConsignment::CC_NL)
-        ) {
+        $localIsNlOrBe = in_array($local, [AbstractConsignment::CC_BE, AbstractConsignment::CC_NL]);
+        $destinationIsNl = AbstractConsignment::CC_NL;
+        $destinationIsBe = AbstractConsignment::CC_BE;
```

Comment:
Als je `is` gebruikt in een $varIs dan verwacht ik dat het een boolean is, maar nu is het een string.
Comment van Edie klopt wel.

Dus `$destinationIsNl = $destination === AbstractConsignment::CC_NL`

Reply:
O, helemaal overheen gekeken. Voeg het meteen toe!